mavros_extras/mavros_plugins.xml

```diff
@@ -69,6 +69,12 @@ Sends fake GPS from local position estimation source data (motion capture,
 vision) to FCU - processed in HIL mode or out of it if parameter MAV_USEHILGPS
 is set on PX4 Pro Autopilot Firmware; Ardupilot Firmware already supports it
 without a flag set.</description>
+  </class>
+  <class name="gimbal_control" type="mavros::plugin::PluginFactoryTemplate&lt;mavros::extra_plugins::GimbalControlPlugin&gt;" base_class_type="mavros::plugin::PluginFactory">
+    <description>@brief Gimbal Control Plugin
+@plugin gimbal_control
+
+Adds support for Mavlink Gimbal Protocol v2.</description>
```

Comment:
Original comment by "vooon": 
"I think you haven't used cog.py to re-generate that file?" 

I am not sure how to run it so it effects this file? I can the `./mavros/tools/cogall.sh` but it did not seem to regenerate this file. Could you point me in the right direction to figure out how to regenerate it?

Reply:
https://github.com/mavlink/mavros/blob/ros2/cog-requirements.txt

Cogall depends on installed cog.py, probbly in today's world that be easier with pipx, but i haven't exact directions for that.

So in general, you must do like that (beware, not tested!):
```
python3 -m venv ~/ros/.venv
source ~/ros/.venv
pip3 install -r cog-requirements.txt
pip3 install pymavlink
ln mavros_cog.py ~/ros/.venv/lib/python3.11/site-packages/
```

Then
```
cog -cr mavros_extras/mavros_plugins.xml
```